Makefile

```diff
@@ -9,10 +9,21 @@ PORT ?= 9292
 
 all: check
 
-setup:
-	bundle check || bundle install
+.env:
 	cp .env.example .env
 
+public/vendor:
+	mkdir -p public/vendor
+
+install_dependencies:
+	bundle check || bundle install
+	yarn install
+
+copy_vendor: public/vendor
+	cp -R node_modules/uswds/dist public/vendor/uswds
```

Comment:
This is needed before deploying.

Can we update our handbook to describe that this step is needed? Or maybe add a `make prep_deploy` that calls this and `rake deploy:json` too? https://handbook.login.gov/articles/appdev-cloud-gov-deploy.html#deploying

Reply:
Handbook documentation is updated in https://github.com/18F/identity-handbook/pull/100